bigbluebutton-html5/imports/api/common/server/helpers.js

```diff
@@ -45,3 +45,20 @@ export const extractCredentials = (credentials) => {
   const requesterUserId = credentialsArray[1];
   return { meetingId, requesterUserId };
 };
+
+export const publicationSafeGuard = function (fn, self) {
+  let stopped = false;
+  const periodicCheck = function () {
+    if (stopped) return;
+    if (!fn()) {
+      self.added(self._name, 'publication-stop-marker', { stopped: true });
```

Comment:
Please add a log reporting that a "server->client" publication stop happened.

Reply:
I added a log to safeguard and check function due they are abstract and don't have the information of each other

Reply:
Great, thank you.